FreshRSS/FreshRSS
# improved: add timeout of "load more articles"

I had very slow internet yesterday. So it was very difficult to load more articles and I saw very often for a long time the "loading circle" at the bottom of the articles list. I did not knew whether it is still loading or already failed.

Changes proposed in this pull request:

- add a timeout of 30 seconds to load the next articles
- if timeout is reached: show the "Load more articles" button again (instead of the loading circle gif)

How to test the feature manually:

1. block your internet connection or slow it down (f.e. with the Firefox browser tool that tests the screen resolution)
2. scroll down the articles list till the next articles should load
3. wait for timeout
4. the loading circle will disappear and the loading button will be shown

Pull request checklist:
